### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ Deploy Dacpac Settings_Screenreader: After opening the screen NVDA is not announcing the title of the screen.

### ISSUE THREAD

**Issue by mspatil110** [microsoft/azuredatastudio#6824]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tools Name: Accessibility insight tool
MAS References: MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select first radio button.
9. Launch NVDA.
10. Navigate till next button & hit enter.
11. Observed that NVDA is announcing or not the title of screen after opening the screen.

**Actual:**
After opening the screen Select Deploy Dacpac Settings NVDA is not announcing the title of the screen.

**This issue has observed through out the application.**

**Expected:**
After opening the screen Select Deploy Dacpac Settings NVDA should announce the title of the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will face difficulty to understand screen without title.

**MAS Reference:**

MAS 1.3.1- https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4} 

**Attachment Reference:**
![6824_A11y_AzureDataStudio(Schema,Dacpac,Backup)_ Deploy Dacpac Settings_Screenreader](https://user-images.githubusercontent.com/51433293/63336704-37612900-c35d-11e9-8e73-82cd9cb4b5c8.png)
